main.tf

```diff
@@ -764,3 +764,52 @@ resource "aws_lb_listener_certificate" "https_listener" {
   listener_arn    = aws_lb_listener.frontend_https[var.extra_ssl_certs[count.index]["https_listener_index"]].arn
   certificate_arn = var.extra_ssl_certs[count.index]["certificate_arn"]
 }
+
+################################################################################
+# Security Group
+################################################################################
+
+locals {
+  create_security_group = local.create_lb && var.create_security_group
+  security_group_name   = try(coalesce(var.security_group_name, var.name, var.name_prefix), "")
+}
+
+data "aws_subnet" "this" {
+  count = local.create_security_group ? 1 : 0
+
+  id = element(var.subnets, 0)
+}
+
+resource "aws_security_group" "this" {
+  count = local.create_security_group ? 1 : 0
+
+  name        = var.security_group_use_name_prefix ? null : local.security_group_name
+  name_prefix = var.security_group_use_name_prefix ? "${local.security_group_name}-" : null
+  description = var.security_group_description
+  vpc_id      = data.aws_subnet.this[0].vpc_id
+
+  tags = merge(var.tags, var.security_group_tags)
```

Comment:
Let's add `Name` tag at the end, as we do for other resources already (like [this one](https://github.com/terraform-aws-modules/terraform-aws-alb/pull/273/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR115)):
```
    {
      "Name" = ... (something like `var.security_group_use_name_prefix ? null : local.security_group_name` maybe)
    },
```

Reply:
added in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/273/commits/0ad1649bf8035fe94603fac53ae1b21bda6501ba